index.js

```diff
@@ -45,20 +49,36 @@ var Input = React.createClass({
 
     var classes = this.getInitialClasses();
 
+    var inputProps = {};
+    if (typeof this.props.value === 'string') {
+      // make input controlled
+      inputProps['value'] = this.props.value;
+    }
+
     if (this.state.isActive) {
       classes.push('is-active');
     }
 
     return <div className={ classes.join(' ') }>
       <label>{ this.props.label }</label>
-      <input type={ type } placeholder={ this.props.label } onChange={ this.onChange } />
+      <input type={ type } placeholder={ this.props.label } onChange={ this.onChange } {...inputProps} />
     </div>;
   }
 });
 
+Input.propTypes = {
+  label: React.PropTypes.string.isRequired,
+  type: React.PropTypes.string,
+  className: React.PropTypes.className,
+  onChange: React.PropTypes.func,
+  value: React.PropTypes.string,
+}
+
 
 /**
  * Expose component
  */
 
-module.exports = Input;
+if (module && module.exports) {
```

Comment:
As I was hacking on stuff to make the `demo.html` work it seemed as if the transpiler errored (silently) because `window.module` is `undefined`. I can remove it if you want?